lib/jekyll-import/importers/drupal7.rb

```diff
@@ -72,10 +74,11 @@ def self.process(options)
           # Get the relevant fields as a hash, delete empty fields and convert
           # to YAML for the header
           data = {
-             'layout' => 'post',
-             'title' => title.to_s,
-             'created' => created,
-           }.delete_if { |k,v| v.nil? || v == ''}.to_yaml
+            'layout' => 'post',
+            'title' => title.to_s,
+            'created' => created,
+            'excerpt' => summary
```

Comment:
It can't hurt; but the `body_summary` field from the database is of type `LONGTEXT`, so I'm not sure it's necessary (especially considering `created` isn't being cast to an integer—it's just being trusted as a valid integer value).